src/Microsoft.AspNet.Authentication.OpenIdConnect/DefaultIdentityModelLoggingListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Authentication.OpenIdConnect
+{
+    /// <summary>
+    /// Default logging event listener for wilson event source. It logs the data to the target where other Asp.Net logs go.
+    /// </summary>
+    public class DefaultIdentityModelLoggingListener : EventListener
```

Comment:
Despite its name, aspnet/Logging is not an ASP.NET component - which explains why it has a neutral namespace: "Microsoft.Framework.Logging".